lib/WasmReader/WasmParseTree.h

```diff
@@ -15,8 +15,10 @@ namespace Wasm
             Void,
 #define WASM_LOCALTYPE(token, name) token,
 #include "WasmKeywords.h"
-            Limit
+            Limit,
+            Unreachable
```

Comment:
Yeah, I think there's going to need a good revamp of that enum.
I will try to do some cleanup of this enum in a later change